src/main/java/raisetech/StudentManagement/Student.java

```diff
@@ -0,0 +1,19 @@
+package raisetech.StudentManagement;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+
+public class Student {
```

Comment:
クラスレベルのアノテーションはクラスメイト空行をあけないほうが読みやすいです。
```suggestion
@Getter
@Setter
public class Student {
```